src/values.rs

```diff
@@ -76,6 +76,10 @@ pub enum Value {
         #[serde(with = "range_serde")]
         range: Range,
     },
+    IntRange {
+        x: Box<Value>,
+        y: Box<Value>,
+    },
```

Comment:
This avoids an allocation.
```suggestion
    IntRange(Box<Range<Value>>),
```